src/glacier_upload/initiate_job.py

```diff
@@ -50,7 +50,7 @@ def init_inventory_retrieval(vault_name, format, description):
 )
 @click.option("-d", "--description", help="Description of this job (optional)")
 def init_archive_retrieval_command(vault_name, archive_id, description):
-    return init_archive_retrieval_command(vault_name, archive_id, description)
+    return init_archive_retrieval(vault_name, archive_id, description)
```

Comment:
Is this an intentional change? The command in `setup.py` appears to still point to `init_inventory_retrieval_command`

Reply:
Also yes, because the function itself named 
`def init_archive_retrieval(vault_name, archive_id, description):`
Without these changes I also get an exception